core/src/main/scala/stryker4s/run/EtaCalculator.scala

```diff
@@ -0,0 +1,43 @@
+package stryker4s.run
+
+import grizzled.slf4j.Logging
+
+import scala.concurrent.duration._
+import language.postfixOps
+
+class EtaCalculator(amountOfMutants: Int) extends Logging {
```

Comment:
I like the solution, but I think I would prefer if the EtaCalculator only calculated the ETA. Then the result could be used in different ways. Now it both times and logs

Reply:
I changed it slightly to just return the result of the function and the estimate. Maybe that's better?